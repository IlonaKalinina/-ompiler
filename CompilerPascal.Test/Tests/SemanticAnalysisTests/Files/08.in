program Test_08;
var
a : integer;
begin
a := 'hello';
end.