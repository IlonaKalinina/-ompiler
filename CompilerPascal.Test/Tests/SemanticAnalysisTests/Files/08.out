program Test_08
├───var
│    └─── :
│         ├─── a
│         └─── integer
│    
begin
├─── :=
│    ├─── a
│    └─── hello
└─── end

Symbol Tables:
builtins:
	integer: SymInteger
	real: SymReal
	string: SymString
	write: SymProc
	read: SymProc
globals:
	a: SymbolVar
